.editorconfig

```diff
@@ -165,32 +172,63 @@ csharp_space_between_square_brackets = false
 csharp_preserve_single_line_statements = false
 csharp_preserve_single_line_blocks = true
 
+# ----------------------------------------------------------------------------------------------------------------------
+# Naming conventions
+# ----------------------------------------------------------------------------------------------------------------------
+
 # ORDERING DOES MATTER!!!
 # Naming conventions should be ordered from most-specific to least-specific in the .editorconfig file.
 # The first rule encountered that can be applied is the only rule that is applied.
 
 [*.{cs,vb}]
+
+# Naming rules
+
+dotnet_naming_rule.interface_must_start_with_I.severity = warning
+dotnet_naming_rule.interface_must_start_with_I.symbols = interface_types
+dotnet_naming_rule.interface_must_start_with_I.style = I_style
+
+dotnet_naming_rule.variables_must_be_camel_style.severity = warning
+dotnet_naming_rule.variables_must_be_camel_style.symbols = parameter_types
+dotnet_naming_rule.variables_must_be_camel_style.style = camel_style
+
+dotnet_naming_rule.types_should_be_pascal_case.severity = suggestion
```

Comment:
I think this (types_should_be_pascal_case) should also be a warning. Do you have some exceptions in mind?

Reply:
I was probably just tired by line 195 🙂